tests/php/FileSubsitesTest.php

```diff
@@ -1,43 +1,53 @@
 <?php
 
+namespace SilverStripe\Subsites\Tests;
+
+use SilverStripe\Assets\File;
+use SilverStripe\Assets\Folder;
+use SilverStripe\Forms\FieldList;
+use SilverStripe\Security\Member;
+use SilverStripe\Subsites\Extensions\FileSubsites;
+use SilverStripe\Subsites\Model\Subsite;
+
 class FileSubsitesTest extends BaseSubsiteTest
 {
-    public static $fixture_file = 'subsites/tests/SubsiteTest.yml';
+
+    public static $fixture_file = 'subsites/tests/php/SubsiteTest.yml';
 
     /**
      * Disable other file extensions
      *
      * @var array
      */
-	protected $illegalExtensions = array(
-        'File' => array(
+    protected $illegalExtensions = [
+        'File' => [
             'SecureFileExtension',
             'VersionedFileExtension'
-        ),
-		'SiteTree' => array(
-			'Translatable',
-		)
-	);
+        ],
+        'SiteTree' => [
```

Comment:
`SiteTree::class`, and `File::class` above, as well as the extensions - `SecureFileExtension::class` etc